### PR TITLE
plugins/inventory/cobbler: Add exclude/include_mgmt_classes

### PR DESCRIPTION
**Pull request by opoplawski**
##### SUMMARY
Add exclude/include_mgmt_classes options to be able to include or exclude hosts based on their management class.

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
plugins/inventory/cobbler.py

### CODE REVIEW

File: changelogs/fragments/7184-cobbler-mgmt-classes.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - cobbler inventory plugin - add ``exclude_mgmt_classes`` and ``include_mgmt_classes`` option to exclude or include hosts based on management classes (https://github.com/ansible-collections/community.general/pull/7184).
```
Comment (felixfontein) [inline changelogs/fragments/7184-cobbler-mgmt-classes.yml:2]:
```suggestion
  - cobbler inventory plugin - add ``exclude_mgmt_classes`` and ``include_mgmt_classes`` options to exclude or include hosts based on management classes (https://github.com/ansible-collections/community.general/pull/7184).
```

File: plugins/inventory/cobbler.py [hunk 2]
```diff
@@ -42,13 +42,25 @@
         description: Fallback to cached results if connection to cobbler fails.
         type: boolean
         default: false
+      exclude_mgmt_classes:
+        description: Management classes to exclude from inventory
```
Comment (felixfontein) [inline plugins/inventory/cobbler.py:46]:
```suggestion
        description: Management classes to exclude from inventory.
```

File: plugins/inventory/cobbler.py [hunk 3]
```diff
@@ -42,13 +42,25 @@
...
+        version_added: 7.4.0
       exclude_profiles:
         description:
           - Profiles to exclude from inventory.
           - Ignored if O(include_profiles) is specified.
         type: list
         default: []
         elements: str
+      include_mgmt_classes:
+        description: Management classes to include from inventory
```
Comment (felixfontein) [inline plugins/inventory/cobbler.py:59]:
```suggestion
        description: Management classes to include from inventory.
```

**Review by felixfontein — COMMENTED** [review]
Thanks for your contribution!

File: changelogs/fragments/7184-cobbler-mgmt-classes.yml (hunk 1 above)
Comment (opoplawski) [reply changelogs/fragments/7184-cobbler-mgmt-classes.yml]: Fixed

File: plugins/inventory/cobbler.py (hunk 3 above)
Comment (opoplawski) [reply plugins/inventory/cobbler.py]: Fixed

File: plugins/inventory/cobbler.py (hunk 2 above)
Comment (opoplawski) [reply plugins/inventory/cobbler.py]: Fixed

**Review by felixfontein — APPROVED** [review]

**Pull request merged**